subsys/bluetooth/Kconfig.scan

```diff
@@ -116,6 +116,73 @@ config BT_SCAN_MANUFACTURER_DATA_CNT
 
 endif
 
+config BT_SCAN_CONNECTION_ATTEMPTS_FILTER
+	bool "Connection attempts filter"
+	help
+	 Connection attempts filter. This filter limits the number of attempts
+	 to connect to the device in case of fail.
```

Comment:
I would remove `in case of fail`. It is no longer true, because we count all the disconnection.

Reply:
Removed